guides/doc-Provisioning_Guide/topics/Virt-VMWare.adoc

```diff
@@ -219,6 +219,161 @@ Create the host from an image with the `hammer host create` command and include
 
 For more information about additional host creation parameters for this compute resource, see xref:CLI_Params[].
 
+[[Provisioning_Virtual_Machines_in_VMware_vSphere-Provisioning_with_cloudinit_and_userdata_templates]]
+=== Using the VMware vSphere cloud-init and userdata Templates for Provisioning
+
+You can use VMware with the `Cloud-init` and `userdata` templates to insert user data into the new virtual machine, to make further VMware customization, and to enable the VMware-hosted virtual machine to call back to {Project} at intervals.
```

Comment:
Why cloud-init is capital letter while userdata not? They are both the very same template kinds, just a different name for the similar thing.

Also can you explain "at intervals" to me? I don't get this.

Reply:
Intervals is a set amount of time. So, every five minutes could also be written as "at five-minute intervals*. To be honest, I don't remember why I wrote it like this, but from what is written here, I would assume that the VM pings Foreman/Satellite at a set frequency of time. If this is not true, I can edit.

Reply:
The thing is I don't remember anything that occurs in any interval when it comes to VMWare provisioning. Maybe cloud-init software has some intervals between "call home" attempts, I am not sure if that's too relevant in this documentation tho.